tests/lib/bin/mountinfo-tool

```diff
@@ -749,7 +749,8 @@ Known attributes, applicable for both filtering and display.
         raise SystemExit(
             "--one requires exactly one match, found {}".format(len(entries))
         )
-
+    if len(entries) == 0:
```

Comment:
Could you please add a comment here of what's the use case (i.e. what you explained in PR description). As as it may look like an error case that's missing a proper message.

Reply:
Sure, the idea is that it allows you to write scripts like:
```
if mountinfo-tool /some/path; then
   # stuff is mounted, do something
fi
```

I'll add a comment in the code.

Reply:
Yes, sure, sorry for not being clear: I understand this change (PR description is clear), it is just about adding a comment for anyone who looks at this chunk of code later.